### DISCUSSION TITLE
Out of memory

### DISCUSSION THREAD

**Discussion by MarcosDLR** [open-wa/wa-automate-nodejs#2643] [General]
Hello.

After the browser has been open for several days, this error appears. If anyone knows how to prevent this please let me know.
![outOfMemory](https://user-images.githubusercontent.com/47016656/160405315-c6d04ca2-6f69-43e1-9ab0-5a6748f3056e.jpeg)

**Comment by smashah** [comment]
@MarcosDLR what memory management strategies have you tried so far?

**Comment by MarcosDLR** [comment]
I have used the refresh method every 24 hours, but this method sometimes messes up the page.

**Comment by smashah** [comment]
@MarcosDLR have you tried using pm2 to restart the process based on memory usage?

**Comment by MarcosDLR** [comment]
I haven't tried it yet.

Is there any way to increase the maximum memory allowed to the browser by pupperter ?

**Comment by gnadelwartz** [comment]
@MarcosDLR I restart wa-automate daily to avoid problems like this, chromium is a memory hog
